scripts/jmx_files/CSM_Create_N_CSM_Users.jmx

```diff
@@ -38,7 +38,7 @@
         <stringProp name="shareMode">shareMode.all</stringProp>
       </CSVDataSet>
       <hashTree/>
-      <ResultCollector guiclass="ViewResultsFullVisualizer" testclass="ResultCollector" testname="View Results Tree" enabled="true">
```

Comment:
While checking in , we should disable the View results tree. It is only for debugging

Reply:
updated